ddp-workspace/projects/ddp-pancan/src/app/components/welcome/stay-informed-section/stay-informed-section.component.ts

```diff
@@ -0,0 +1,31 @@
+import { ChangeDetectionStrategy, Component, Inject, Input, OnInit } from '@angular/core';
+import { AppRoutes } from '../../app-routes';
+import { CommunicationService, ToolkitConfigurationService } from 'toolkit';
+
+@Component({
+    selector: 'app-stay-informed-section',
+    templateUrl: './stay-informed-section.component.html',
+    styleUrls: ['./stay-informed-section.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class StayInformedSectionComponent implements OnInit {
+    @Input() title: string;
+    @Input() btnText: string;
+    readonly AppRoutes = AppRoutes;
+    twitterUrl: string;
+    facebookUrl: string;
```

Comment:
#minor you can make them readonly and assign the values in constructor, not in ngOnInit

Reply:
fixed.